src/main/java/algorithms/curated170/medium/campusbikes/CampusBikesBucketSort.java

```diff
@@ -0,0 +1,58 @@
+package algorithms.curated170.medium;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class CampusBikesBucketSort {
+
+      int w;
+      List<int[]>[] buckets = new List[2001];
+
+      public int[] assignBikes(int[][] workers, int[][] bikes) {
+            
+            w = workers.length;
```

Comment:
As there may be more bikes then workers, we need to have variable for bikes' size also.
```
            workerSize = workers.length;
            bikeSize = bikes.length;
```